webpack.config.js

```diff
@@ -37,6 +38,9 @@ const webpackConfig = {
 		chunkFilename: '[name].[chunkhash].js', // ditto
 		devtoolModuleFilenameTemplate: 'app:///[resource-path]'
 	},
+	postcss() {
```

Comment:
Yep, we migrated that in Delphin:
https://github.com/Automattic/delphin/blob/master/webpack.base.config.js#L64